chaste_codegen/chaste_model.py

```diff
@@ -400,6 +402,16 @@ def _convert_membrane_stimulus_current(self):
         return membrane_stimulus_current_converted, \
             (self._membrane_stimulus_current_orig.units, membrane_stimulus_current_converted.units)
 
+    def _convert_other_currents(self):
+        """ Try to convert other currents (apart from stimulus current) to uA_per_cm2 """
```

Comment:
This looks good. Maybe add to the docstring that only variables in units convertible to Chaste's current units are converted, to allow for 'current-related' variables in the ontology that aren't themselves currents but might be included within the superclass.